frontend/src/metabase/dashboard/actions/data-fetching-typed.ts

```diff
@@ -210,3 +250,329 @@ export const SET_SHOW_LOADING_COMPLETE_FAVICON =
 export const setShowLoadingCompleteFavicon = createAction<boolean>(
   SET_SHOW_LOADING_COMPLETE_FAVICON,
 );
+
```

Comment:
I just moved these from `data-fetching` and added types. 

If the string didn't have an associated action, I created one with `createAction` and added types. I then used that action in the reducer rather than the string itself